### ISSUE TITLE
az webapp list-runtimes error

### ISSUE THREAD

**Issue by sweetmandoo** [Azure/azure-cli#12227]
## Describe the bug

**Command Name**

az webapp list-runtimes

**Errors:**

The command failed with an unexpected error. Here is the traceback:

'bool' object has no attribute 'provider'
Traceback (most recent call last):
  File "/usr/local/Cellar/azure-cli/2.0.81/libexec/lib/python3.8/site-packages/knack/cli.py", line 206, in invoke
    cmd_result = self.invocation.execute(args)
  File "/usr/local/Cellar/azure-cli/2.0.81/libexec/lib/python3.8/site-packages/azure/cli/core/commands/__init__.py", line 608, in execute
    raise ex
  File "/usr/local/Cellar/azure-cli/2.0.81/libexec/lib/python3.8/site-packages/azure/cli/core/commands/__init__.py", line 666, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/usr/local/Cellar/azure-cli/2.0.81/libexec/lib/python3.8/site-packages/azure/cli/core/commands/__init__.py", line 659, in _run_job
    six.reraise(*sys.exc_info())
  File "/usr/local/Cellar/azure-cli/2.0.81/libexec/lib/python3.8/site-packages/six.py", line 703, in reraise
    raise value
  File "/usr/local/Cellar/azure-cli/2.0.81/libexec/lib/python3.8/site-packages/azure/cli/core/commands/__init__.py", line 636, in _run_job
    result = cmd_copy(params)
  File "/usr/local/Cellar/azure-cli/2.0.81/libexec/lib/python3.8/site-packages/azure/cli/core/commands/__init__.py", line 306, in __call__
    return self.handler(*args, **kwargs)
  File "/usr/local/Cellar/azure-cli/2.0.81/libexec/lib/python3.8/site-packages/azure/cli/core/__init__.py", line 493, in default_command_handler
    return op(**command_args)
  File "/usr/local/Cellar/azure-cli/2.0.81/libexec/lib/python3.8/site-packages/azure/cli/command_modules/appservice/custom.py", line 704, in list_runtimes
    return [s['displayName'] for s in runtime_helper.stacks]
  File "/usr/local/Cellar/azure-cli/2.0.81/libexec/lib/python3.8/site-packages/azure/cli/command_modules/appservice/custom.py", line 2159, in stacks
    self._load_stacks()
  File "/usr/local/Cellar/azure-cli/2.0.81/libexec/lib/python3.8/site-packages/azure/cli/command_modules/appservice/custom.py", line 2180, in _load_stacks
    raw_stacks = self._client.provider.get_available_stacks(os_type_selected=os_type, raw=True)
AttributeError: 'bool' object has no attribute 'provider'

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- none

## Environment Summary

macOS-10.15.3-x86_64-i386-64bit
Python 3.8.1
Shell: bash

azure-cli 2.0.81

```
## Additional Context

<!--Please don't remove this:-->
<!--auto-generated-->

**Comment by arrownj** [comment]
@sweetmandoo , thanks for reporting, we are looking at it.

**Comment by panchagnula** [comment]
This now fixed with the latest version of CLI 2.2.0 please re-try. Thanks!
